app/styles/_projects.less

```diff
@@ -19,6 +19,13 @@
   }
 }
 
```

Comment:
Fix looks good to me. I have a PR to common that we will need tomorrow anyway, if that makes you feel better about migrating it up.